src/index.html

```
@@ -0,0 +1,53 @@
+    <main>
+      <section class="input__section">
+        <div class="input__inner__box">
+          <h2 class="input__heading">Enter Input to start</h2>
+          <form id="user_input">
+            <div class="input__fields">
+              <div>
+                <label for="floorInput" class="input_labels"
+                  >Enter The Number Of Floors</label
+                >
+                <input
+                  type="number"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)